set.mm

```diff
@@ -523993,12 +524024,6 @@ which requires fewer axioms (i.e., removes dependency on ~ ax-6 ,
       ( wtru crab wceq cv wcel wb nfrab1 nfcv cleqf tru rabid mpbiran2 mpgbir )
       CABDZBEAFZPGZQBGZHAAPBCABIABJKRSCLCABMNO $.
 
-    $( Alternate proof of ~ bj-rabtr .  (Contributed by BJ, 22-Apr-2019.)
-       (Proof modification is discouraged.)  (New usage is discouraged.) $)
-    bj-rabtrALTALT $p |- { x e. A | T. } = A $=
-      ( wtru crab ssrab2 wss wral ssid tru rgenw ssrab mpbir2an eqssi ) CABDZBC
-      ABEBNFBBFCABGBHCABIJCABBKLM $.
-
```

Comment:
Thanks. bj-rabtr may have changed to align with bj-rabtrALTALT when the definition of TRUE was changed.